### PR TITLE
Fixed the Multi Instance Display of Envision

### PR DESCRIPTION
**Pull request by RutvikGupta**
Fixes Issue #784. Edited some changes of PR #545 so that envision does not reconnects to a running simulation when switched to "All Simulation" or multi-instance display.

### CODE REVIEW

File: envision/web/src/components/app.js [hunk 1]
```diff
@@ -67,20 +67,20 @@ function App({ client }) {
...
       setSimulationIds(ids);
-    };
+    })();
 
     // checks if there is new simulation running every 3 seconds.
-    const interval = setInterval(fetchRunningSim, 3000);
-    return () => clearInterval(interval);
-  }, [matchedSimulationId]);
+    // const interval = setInterval(fetchRunningSim, 3000);
+    // return () => clearInterval(interval);
```
Comment (RutvikGupta) [inline envision/web/src/components/app.js:82]: I had to comment out these two lines since they kept on reconnecting the server to a running Simulation when you switch to "All Simulation" or multi-instance display page. The downside now is that the user needs to reload the server every time a new Simulation is added to the server port.

Comment (RutvikGupta) [reply envision/web/src/components/app.js:82]: One thing we can do is to check if there is a change in the number of ids. If there is, we can call the `fetchRunningSim`, else we don't set the interval for `fetchRunningSim`.

Comment (RutvikGupta) [reply envision/web/src/components/app.js:82]: Ok , I tried this and I don't think this can work.

Comment (RutvikGupta) [reply envision/web/src/components/app.js]: UPDATE: This has been fixed now!

**Comment by Gamenot** [comment]
I have tested this out, it appears to be working. I think there are still a few problems remaining here but it is in much better shape than before.

**Review by Gamenot — APPROVED** [review]

File: envision/web/src/components/app.js [hunk 2]
```diff
@@ -61,26 +61,29 @@ function App({ client }) {
   const recorderRef = useRef(null);
   const { addToast } = useToasts();
   const history = useHistory();
+  const [totalSimIds, setTotalSimIds] = useState(0);
+
+  let getSimIdsLength = async () => {
+    let ids = await client.fetchSimulationIds();
+    setTotalSimIds(ids.length);
+  };
+  setInterval(getSimIdsLength, 3000);
```
Comment (Gamenot) [inline envision/web/src/components/app.js:70]: I believe this works better. The one remaining part of it is that we will need to fix the automatic navigation to a simulation page that occurs when the number of simulation IDs change but I also think we can leave this to a different PR.

Comment (RutvikGupta) [reply envision/web/src/components/app.js:70]: Hmm, I when I try, it doesn't navigates to simulation page when no. of sim IDs changes but remains on the simulation you were originally on.

Comment (RutvikGupta) [reply envision/web/src/components/app.js:70]: Oh, you meant when you are on "All Simulation" page. Yeah well that makes sense because for a new simulation ID it pushes it to the `history` and the server reconnects envision to that simulation if you are not connected to any simulation i.e. if you are on the multi-instance display.

Comment (Gamenot) [reply envision/web/src/components/app.js:70]: #961

File: Makefile [hunk 3]
```diff
@@ -130,7 +130,7 @@ clean:
 .PHONY: format
 format:
 	# pip install isort==5.7.0
-	isort -m VERTICAL_HANGING_INDENT --skip-gitignore --ac --tc --profile black ./benchmark/ ./cli ./envision ./examples/ ./extras/ ./scenarios/ ./smarts ./ultra ./zoo
+	isort -m VERTICAL_HANGING_INDENT --skip-gitignore --ac --tc --profile black ./baselines/benchmark/ ./cli ./envision ./examples/ ./utils/ ./scenarios/ ./smarts ./ultra ./zoo
```
Comment (Adaickalavan) [inline Makefile:133]: Is it `./baselines/marl_benchmark` ?

Comment (RutvikGupta) [reply Makefile]: Yes. Fixed it.

**Review by Adaickalavan — APPROVED** [review]
